schemas/context/profile-lead-crm.schema.json

```diff
@@ -0,0 +1,172 @@
+          "title": "Lead Source Type",
+          "description": "Lead source type.",
+          "type": "string"
+        },		
+        "xdm:converted": {
+          "title": "Converted Indicator",
+          "description": "Indicates whether the Lead was successfully converted or not.",
+          "type": "boolean"
+        },
+        "xdm:convertedAccountID": {
+          "title": "Converted Account ID",
+          "description": "Account Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:convertedContactID": {
+          "title": "Converted Contact ID",
+          "description": "Contact Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:convertedDate": {
+          "title": "Converted Date",
+          "description": "Date of conversion if the lead was successfully converted.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:convertedOpportunityID": {
+          "title": "Converted Opportunity ID",
+          "description": "Opportunity Id if the lead was successfully converted.",
+          "type": "string"
+        },
+        "xdm:deleted": {
+          "title": "Deleted Indicator",
+          "description": "Indicator whether the lead was deleted.",
+          "type": "boolean"
+        },
+        "xdm:leadDescription": {
+          "title": "Lead Description",
+          "description": "Lead description.",
+          "type": "string"
+        },
+        "xdm:emailBouncedDate": {
+          "title": "Email Bounced Date",
+          "description": "If bounce management is activated and an email is sent to a contact, provides the date when the email was bounced.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:emailBouncedReason": {
+          "title": "Email Bounced Reason",
+          "description": "If bounce management is activated and an email is sent to a contact, provides the reason for bounced email.",
+          "type": "string"
+        },
+        "xdm:jigsawContactID": {
+          "title": "Jigsaw Contact Id",
+          "description": "Lead’s Data.com ID if available.",
+          "type": "string"
+        },
+        "xdm:lastTransferDate": {
+          "title": "Last Transfer Date",
+          "description": "Last transfer date.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:ownerID": {
+          "title": "Owner ID",
+          "description": "Owner id.",
+          "type": "string"
+        },
+        "xdm:leadSource": {
+          "title": "Lead Source",
+          "description": "Field recording what source the lead originated from.",
+          "type": "string"
+        },
+        "xdm:leadStatus": {
+          "title": "Lead Status",
+          "description": "Field recording the current marketing/sales status of the lead.",
+          "type": "string"
+        },
+        "xdm:leadPhotoUrl": {
```

Comment:
Part of data type profile-personal-details, and refer all the data types in the allOF